wiki/simple_vm/workshop.md

```diff
@@ -67,6 +67,12 @@ machines later on, if they lose their private key.
 !!! Tip
     Make sure the admins of your project also have their SSH key set.
 
-##Trouble Shooting
+
+### Access to machines
+
+!!! Info
```

Comment:
There seems to be something missing here?

Reply:
added missing part